src/components/restaurant/RestaurantView.jsx

```diff
@@ -0,0 +1,18 @@
+export default function RestaurantView({
+  name,
+  category,
+  address,
+  onChangeName,
+  onChangeCategory,
+  onChangeAddress,
+  onClickAdd,
+}) {
+  return (
+    <form aria-label="form">
+      <input type="text" aria-label="name" id="name" name="name" placeholder="이름" value={name} onChange={onChangeName} />
```

Comment:
```suggestion
      <input 
        type="text"
        aria-label="name" 
        id="name" 
        name="name" 
        placeholder="이름" 
        value={name} 
        onChange={onChangeName} 
      />
```

저는 속성이 2개 이상일 때는 줄바꿈해주면 코드가독성이 더 좋다고 생각하고 코드를 정리해주는 편이에요.
줄바꿈으로 코드가독성을 높여주는건 어떨까요? 🤔

Reply:
넵 :) 말씀하신대로 진행하겠습니다.